src/PrizeVault.sol

```diff
@@ -284,48 +300,57 @@ contract PrizeVault is TwabERC20, Claimable, IERC4626, ILiquidationSource, Ownab
         }
     }
 
     /// @inheritdoc IERC4626
     /// @dev Considers the uint96 limit on total share supply in the TwabController
+    /// @dev TODO: add reasoning for exclusion of latent balance
+    /// @dev TODO: return 0 if deposits and mints are disabled due to less assets than supply
     function maxDeposit(address) public view returns (uint256) {
         // the vault will never mint more than 1 share per asset, so no need to convert supply buffer to assets
         uint256 _supplyBuffer = type(uint96).max - totalSupply();
-        uint256 _maxAssetDeposit = yieldVault.maxDeposit(address(this));
-        return _supplyBuffer < _maxAssetDeposit ? _supplyBuffer : _maxAssetDeposit;
+        uint256 _maxDeposit;
+        uint256 _latentBalance = _asset.balanceOf(address(this));
+        uint256 _maxYieldVaultDeposit = yieldVault.maxDeposit(address(this));
+        if (_latentBalance >= _maxYieldVaultDeposit) {
+            return 0;
+        } else {
+            unchecked {
+                _maxDeposit = _maxYieldVaultDeposit - _latentBalance;
+            }
+            return _supplyBuffer < _maxDeposit ? _supplyBuffer : _maxDeposit;
+        }
     }
 
     /// @inheritdoc IERC4626
     /// @dev Considers the uint96 limit on total share supply in the TwabController
-    function maxMint(address) public view returns (uint256) {
-        uint256 _supplyBuffer = type(uint96).max - totalSupply();
-
+    function maxMint(address _owner) public view returns (uint256) {
         // shares represent how many assets an account has deposited, so they are 1:1 on mint
-        uint256 _maxShareMint = yieldVault.maxDeposit(address(this));
-        return _supplyBuffer < _maxShareMint ? _supplyBuffer : _maxShareMint;
+        return maxDeposit(_owner);
     }
 
     /// @inheritdoc IERC4626
+    /// @dev TODO: add reasoning for inclusion of latent balance
     function maxWithdraw(address _owner) public view returns (uint256) {
-        uint256 _maxAssetWithdraw = yieldVault.maxWithdraw(address(this));
+        uint256 _maxWithdraw = yieldVault.maxWithdraw(address(this)) + _asset.balanceOf(address(this));
 
         // the owner may receive less than 1 asset per share, so we must convert their balance here
         uint256 _ownerAssets = convertToAssets(balanceOf(_owner));
-        return _ownerAssets < _maxAssetWithdraw ? _ownerAssets : _maxAssetWithdraw;
+        return _ownerAssets < _maxWithdraw ? _ownerAssets : _maxWithdraw;
     }
 
     /// @inheritdoc IERC4626
+    /// @dev TODO: add reasoning for inclusion of latent balance
     function maxRedeem(address _owner) public view returns (uint256) {
         // the owner will never receive more than 1 asset per share, so no need to convert max withdraw to shares
-        uint256 _maxShareRedemption = yieldVault.maxWithdraw(address(this));
+        uint256 _maxRedeem = yieldVault.maxWithdraw(address(this)) + _asset.balanceOf(address(this));
```

Comment:
Should convert max withdraw assets to shares rounding up to ensure they get the most out of the limits.